docs/hub/repositories.md

```diff
@@ -0,0 +1,212 @@
+
+We also recommend you create a [Hugging Face account](https://huggingface.co/join) which you'll be able to use to create and manage repositories on the Hub.
+
+To be able to push your code to the Hub, you'll need to authenticate somehow. The easiest way to do this is by installing the [`huggingface_hub` CLI](https://huggingface.co/docs/huggingface_hub/index) and running the login command:
+
+```bash
+python -m pip install huggingface_hub
+huggingface-cli login
+```
+
+The content in the **Getting Started** section of this document is also available as a video!
+
+<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/rkCly_cbMBk" title="Managing a repo" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+### Creating a repository
+
+Using the Hub's web interface you can easily create repositories, add files (even large ones!), explore models, visualize diffs, and much more. There are three kinds of repositories on the Hub, and in this guide we'll be creating a **model repository** for demonstration purposes. For information on creating and managing models, datasets, and Spaces, refer to their respective documentation.
+
+1. To create a new repository, visit [huggingface.co/new](http://huggingface.co/new):
+
+![/docs/assets/hub/new_repo.png](/docs/assets/hub/new_repo.png)
+
+2. First, specify the owner of the repository: this can be either you or any of the organizations you’re affiliated with. 
+
+3. Next, enter your model’s name. This will also be the name of the repository. Finally, you can specify whether you want your model to be public or private.
+
+You can leave the *License* field blank for now. To learn about licenses, visit the **Licenses** (TODO: LINK TO LICENSES) section of this document.
+
+After creating your model repository, you should see a page like this:
+
+![/docs/assets/hub/empty_repo.png](/docs/assets/hub/empty_repo.png)
+
+Note that the Hub prompts you to create a *Model Card*, which you can learn about in the **Model Cards documentation** (TODO: LINK). Including a Model Card in your model repo is best practice, but since we're only making a test repo at the moment we can skip this.
+
+
+### Cloning repositories
+
+Downloading repositories to your local machine is called *cloning*. You can use the following commands to load the repo that we made and navigate to it:
+```bash
+git clone https://huggingface.co/<your-username>/<your-model-id>
+cd <your-model-id>
+```
+
+Then, *within the local repo*, you'll need to run the following to set up Git LFS:
+```bash
+git lfs install
+```
+
+### Adding files to a repository
+
+Now's the time 🔥 You can add any files you want to the repository!
```

Comment:
The 🔥  kind of feels like an interruption in the middle of the sentence. No strong preferences though, so feel free to keep it there if you like! 

```suggestion
Now's the time, you can add any files you want to the repository! 🔥
```